01-getting-started/getting-started.Rmd

```diff
@@ -49,20 +49,20 @@ These requirements can be installed by following the instructions at these links
 We recommend installing `devtools` from CRAN (e.g. running `install.packages("devtools")` in R). 
 
 ### Required software:
-* [**R**](https://cran.r-project.org/)  
-* [**RStudio**](https://www.rstudio.com/products/RStudio/) an [integrated development environment](https://en.wikipedia.org/wiki/Integrated_development_environment) for working with R and [R Notebooks](https://bookdown.org/yihui/rmarkdown/notebook.html).  
-* [**Bioconductor**](https://bioconductor.org/install/) for installing packages from this package repository. 
-* [**tidyverse**](https://www.tidyverse.org/) - we opt for using tidyverse packages for handling and cleaning the data. 
+* [**R**](https://cran.r-project.org/)  [@R-base]  
```

Comment:
Can we settle on a style for bullets? You are using `*`, `-` _and_ `+` in different bullet lists in this doc!
```suggestion

* [**R**](https://cran.r-project.org/)  [@R-base]  
```

Reply:
This is because I borrowed these lists from different places!